fix: Stop creating meta files for each new resource #1217 (#1282)
CommunitySolidServer/CommunitySolidServer: RELEASE_NOTES.md

The following changes are relevant for v3 custom configs that replaced certain features.

> - `config/app/variables/cli.json` was changed to support the new `YargsCliExtractor` format.
> 
> The content-length parser (`config/ldp/metadata-parser/parsers/content-length.json`) has been removed from
> the default config as the metadata it generates is only used in configurations that enforce disk quotas.
> The relevant quota configurations (`config/storage/backend/global-quota-file.json` and 
> `config/storage/backend/pod-quota-file.json`) have been modified to include the content-length parser.

Reviewer: ```suggestion
- `config/app/variables/cli.json` was changed to support the new `YargsCliExtractor` format.
- The content-length parser has been moved from the default configuration to the quota configurations.
   - `/ldp/metadata-parser/default.json`
   - `/storage/backend/*-quota-file.json`
   - `/storage/backend/quota/quota-file.json`
```
To stay more consistent with the previous entries.
